extensions/amp-analytics/0.1/vendors.js

```diff
@@ -59,6 +64,17 @@ export const ANALYTICS_CONFIG = {
           'st=${socialActionTarget}'
     },
     'optout': '_gaUserPrefs.ioo'
+  },
+
+  'parsely': {
+    'host': 'srv.pixel.parsely.com',
+    'method': 'GET',
```

Comment:
Is this intended to set the `transport` value?

https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/amp-analytics.md#transport